testsupport/cleanup/clean.go

```diff
@@ -225,14 +225,14 @@ func (c *cleanTask) verifySpaceDeleted(isUserSignup bool, userSignup *toolchainv
 				c.t.Logf("deleting also the related Space: %s", userSignup.Status.CompliantUsername)
 				if err := c.client.Delete(context.TODO(), space, propagationPolicyOpts); err != nil {
 					if errors.IsNotFound(err) {
-						c.t.Logf("the related Space: %s is deleted as well", userSignup.Status.CompliantUsername)
+						// c.t.Logf("the related Space: %s is deleted as well", userSignup.Status.CompliantUsername)
 						return true, nil
 					}
 					c.t.Logf("problem with deleting the related Space %s: %s", userSignup.Status.CompliantUsername, err)
 					return false, err
 				}
 			}
-			c.t.Logf("waiting until Space: %s is completely deleted", userSignup.Status.CompliantUsername)
```

Comment:
do you mind restoring also this one?  :)
I used it in the past to check if there was some space stuck in deletion...  

And maybe we can delete the commented lines 218 and 228?